.github/workflows/latest_dependency_checker.yml

```diff
@@ -20,18 +20,13 @@ jobs:
       run: |
         python -m virtualenv venv_core
         source venv_core/bin/activate
-        python -m pip install --upgrade pip
-        python -m pip install -r requirements.txt
-        python -m pip install -r test-requirements.txt
```

Comment:
only dependencies being tracked are `nltk` and `featuretools` which are both regular requirements so not necessary to install the test requirements (which also install the complete requirements)